rslib/src/scheduler/answering/mod.rs

```diff
@@ -417,6 +422,33 @@ impl Collection {
         self.add_tags_to_notes_inner(&[nid], "leech")?;
         Ok(())
     }
+
+    fn set_elapsed_time_equal(&self, answer: &mut CardAnswer, current_state: &mut CardState) {
```

Comment:
Since this only changes `current`, answer should not be mutable. Also, a brief docstring explaining the reason for this function would be good.

Reply:
It simplified the function somewhat but I know that it's bad form. I'll switch it around so we update the answer_state to match the current_state instead of the other way around. That also eliminates the need to mark the current state as mutable.